modules/ROOT/pages/vantagecloud-lake-demo-jupyter-docker.adoc

```diff
@@ -0,0 +1,126 @@
+= Run VantageCloud Lake Demos from a Jupyter notebook in Docker
+:experimental:
+:page-author: Vidhan Bhonsle
+:page-email: vidhan.bhonsle@teradata.com
+:page-revdate: January 10th, 2024
+:description: Run Teradata VantageCloud Lake Demos from Jupyter in a Docker
+:keywords: data warehouses, compute storage separation, teradata, vantage, cloud data platform, business intelligence, enterprise analytics, jupyter, teradatasql, ipython-sql, docker, container, vantagecloud, vantagecloud lake, lake
+
+== Overview
+In this how-to we will go through the steps for connecting to TeradataCloud Lake and run demos from a Jupyter notebook in Docker. 
+
+== Prerequisites
+* https://www.docker.com/products/docker-desktop/[Docker Desktop] installed
+* (Optional) https://git-scm.com/book/en/v2/Getting-Started-Installing-Git[Git] installed
+** Required to download git repo from https://github.com/Teradata/lake-demos.git. You can also download the repo as a .zip if you don't have Git installed locally.
+* A Teradata VantageCloud Lake account login
+** Organization URL and login details from Teradata welcome letter
+* (Optional) IDE of your choice
```

Comment:
Would it not make sense to move the optional items to the end of the list?

Reply:
Moved optional items to the end.